WORKSPACE

```diff
@@ -1,22 +1,30 @@
 workspace(name = "bazel_skylib")
 
-load(":workspace.bzl", "bazel_skylib_workspace")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+http_archive(
+    name = "bazel_federation",
+    url = "https://github.com/bazelbuild/bazel-federation/archive/130c84ec6d60f31b711400e8445a8d0d4a2b5de8.zip",
```

Comment:
First line of federation still says "Experimental: do not depend on it."

Can you clarify how Skylib should use the Federation? Anything we should know?

Reply:
We still have to clean up the federation repo before we can cut the first release. I can share an internal document with you that has more details about the project structure and how to use the federation.